data/gsettings/meson.build

```diff
@@ -1,7 +1,14 @@
-configure_file(
-  input: 'org.gnome.GPaste.gschema.xml.in',
+untranslated_gsettings = configure_file(
+  input: 'org.gnome.GPaste.gschema.xml.in.in',
   configuration: conf,
+  output: 'org.gnome.GPaste.gschema.xml.in',
+)
+
+i18n.merge_file(
+  'gsettings',
+  input: untranslated_gsettings,
+  po_dir: gpaste_po_dir,
```

Comment:
Not really related to this, but I'm not quite so sure if GPaste's autotools build system does this correctly as of now. Will need adjustment for this anyway though.

Reply:
Gettext handles this natively, without the need of merging translations in the xml. Works fine with autotools last time I checked

Reply:
AFAICS this is required for at least meson and I just tested autotools and the gschema file isn't translated for me.

Reply:
Autotools gsettings: https://paste.pound-python.org/show/vzfED1mxBvPOJ9zs8Ayj/

Meson gsettings: https://paste.pound-python.org/show/kOq9P5tqOFBVlyJ3ibaw/

Seems to me like translation of that file doesn't work on Autotools.

Reply:
The file itself isn’t translated inline but all users of it (like dconf-editor for example) see the correct translated strings thanks to gettext native support

Reply:
Oh, right - https://developer.gnome.org/gio/stable/GSettings.html

Reply:
I don't think we need to do anything special for it then, since it'll just use what we install in po/meson.build via i18n.gettext()